src/client/src/rt-components/platform/adapters/openfin/openFin.ts

```diff
@@ -108,11 +118,35 @@ export default class OpenFin extends BasePlatformAdapter {
   }
 
   notification = {
-    notify: (message: object) =>
-      new fin.desktop.Notification({
-        url: '/notification',
-        message,
-        timeout: 8000,
-      } as any),
+    notify: (message: object) => {
+      this.createNotification(
+        Object.assign({
```

Comment:
Is there any need for `Object.assign` here?

Reply:
no, I removed it